projects/deep-earth-system-data-lab/collection.json

```diff
@@ -2,7 +2,7 @@
   "type": "Collection",
   "id": "deep-earth-system-data-lab",
   "stac_version": "1.0.0",
-  "description": "The Deep Earth System Data Lab (DeepESDL) delivers a service to the Earth Science community. The lab provides convenient access to relevant data sets, many of them generated in ESA’s science projects, in an analysis-ready format. Moreover, DeepESDL comprises a powerful data science environment with a focus on machine-learning and artificial intelligence workflows. Access the DeepESDL in github: https://github.com/deepesdl",
+  "description": "The Deep Earth System Data Lab (DeepESDL) delivers a service to the Earth Science community. The lab provides convenient access to relevant data sets, many of them generated in ESA\u2019s science projects, in an analysis-ready format. Moreover, DeepESDL comprises a powerful data science environment with a focus on machine-learning and artificial intelligence workflows. Access the DeepESDL in github: https://github.com/deepesdl",
```

Comment:
Could you revert the `\u2019` to `’` please.

Reply:
Done. Please re-review!